project/src/components/places-list/places-list.tsx

```diff
@@ -1,17 +1,28 @@
+import { useState } from 'react';
+
 import CardPlace from '../card-place/card-place';
+import { Offer } from '../../types/offer';
 
 interface PlaceListProps {
-  numberOfPlaces: number;
+  offers: Offer[];
   className: string;
 }
 
-function PlacesList({ numberOfPlaces, className }: PlaceListProps): JSX.Element {
-  const arrayPlaces = [...Array(numberOfPlaces)].map((item, index) => index + 1);
+function PlacesList({ offers, className }: PlaceListProps): JSX.Element {
+  let [offerId, setOfferId] = useState('');
```

Comment:
Начальное состояние null. Тип или number или null
```suggestion
  const [offerId, setOfferId] = useState<number | null>(null);
```